src/http/cookie.cr

```diff
@@ -156,13 +221,30 @@ module HTTP
     # headers in the given `HTTP::Headers`.
     #
     # See `HTTP::Request#cookies` and `HTTP::Client::Response#cookies`.
-    def self.from_headers(headers) : self
+    #
+    # ```crystal
+    # require "http/cookie"
+    #
+    # headers = HTTP::Headers{"Cookie" => "session=god"}
+    # cookies = HTTP::Cookies.from_headers(headers)
+    # cookies # => <HTTP::Cookies @cookies={"session" => <HTTP::Cookie @name="session", @value="god" ...>}>
```

Comment:
You forgot the `#` character denoting the class instance:

```suggestion
    # cookies # => #<HTTP::Cookies @cookies={"session" => #<HTTP::Cookie @name="session", @value="god" ...>}>
```

ditto elsewhere